libs/geography/testing/src/inmemory-backend/geography.service.ts

```diff
@@ -38,4 +40,22 @@ export class DaffInMemoryBackendGeographyService implements InMemoryDbService {
       countries: this.countries
     };
   }
+
+  /**
+   * Responds to GET requests.
+   */
+  get(reqInfo: RequestInfo): any {
```

Comment:
this doesn't seem to behave the same as a real api call. When a DaffCountryLoad action is triggered, no subdivisions are returned, because none exist in the original createDB().

Reply:
good point. I added a line to create country subdivisions in the backend constructor